entrypoint.sh

```diff
@@ -1,19 +1,38 @@
 #!/usr/bin/env bash
 
-# If the CRON variable is empty, phockup gets executed once as command line tool
-if [ -z "$CRON" ]; then
-  phockup "$@"
+if [[ "$PHOCKUP_UID" != "" && "$PHOCKUP_GID" != "" ]]
```

Comment:
Instead of branching this code to two can you extract the user/group ID logic above this code and always use the code that has `$PHOCKUP_UID`, but when the variable is not set use the default user/group ID for the container.
Also please move the `then` to the same line as the `if`